src/sql/sim.sql

```diff
@@ -0,0 +1,33 @@
+CREATE TABLE sim(
+  "iter" INTEGER,
+  "token" TEXT,
+  "all_similarity" REAL,
```

Comment:
similarity between a single token hidden state and the mean of all hidden states of all tokens (the original mean we used)?

i.e. the similarity between an instance of "old" and mean("old")